browser/ui/webui/settings/brave_import_data_handler_mac.mm

```diff
@@ -32,6 +32,26 @@
 
 using content::BrowserThread;
 
+class ScopedOpenFile {
+ public:
+  explicit ScopedOpenFile(const base::FilePath& file_path) {
+    file_ = base::OpenFile(file_path, "r");
```

Comment:
Just trying to open files that need full disk access adds Brave browser to full disk access panel list.